WindowsServerDocs/get-started-19/activation-19.md

```diff
@@ -13,6 +13,15 @@ ms.date: 09/09/2020
 
 >Applies To: Windows Server 2019, Windows Server 2016
 
+> [!NOTE]
+> It is highly recommended to implement ADBA (Active Directory Activation) instead of KMS, as well as migrating existing KMS for modern products to ADBA.<br/>
```

Comment:
Looks like I missed a detail in the explanation for the initialism ADBA:
```suggestion
> It is highly recommended to implement ADBA (Active Directory-Based Activation) instead of KMS, as well as migrating existing KMS for modern products to ADBA.<br/>
```